src/components/Spinner/index.vue

```diff
@@ -0,0 +1,29 @@
+<template src="./index.pug" lang="pug"></template>
+
+<script>
+const Card = () => import("../Card/index");
+const Detail = () => import("../Detail/index");
+
+export default {
+  data: () => ({
+    titleSelected: "",
+    showDetail: false
+  }),
+  components: {
+    Card,
+    Detail
+  },
+  methods: {
+    onShowDetail(value) {
+      console.log("entro", value);
+      this.titleSelected = value;
+      this.showDetail = true;
+    },
+    onCloseDetail() {
+      this.showDetail = false;
+    }
+  }
+};
+</script>
+
+<style src="./index.scss" lang="scss" scoped></style>
```

Comment:
Same for the styles, don't import them, use it inside the `.vue`

Reply:
Why?